packages/server-wallet/tsconfig.json

```diff
@@ -2,14 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
-    // ["es2017", "esnext.bigint", "dom"] needed for https://github.com/bitauth/libauth#typescript-types
-    "lib": [
-      "es5",
-      "es6",
-      "ES2017",
-      "esnext.bigint",
-      "dom"
-    ],
+    "lib": ["ES2019", "dom"],
```

Comment:
I was seeing TS issues with `flatMap` and took the opportunity to clean up the our tsconfig.